docs/general/installation/truenas.md

```diff
@@ -0,0 +1,534 @@
+---
+uid: installation-truenas
+title: TrueNAS SCALE
+description: Install on TrueNAS SCALE.
+sidebar_position: 4
+---
+
+- Jellyfin can be installed on iX-systems' [TrueNAS SCALE](https://www.truenas.com/truenas-scale/).
+- This documentation is written for TrueNAS SCALE v24.10.0 (Electric Eel) or higher.
+- Consider reviewing the [TrueNAS Apps documentation](https://www.truenas.com/docs/truenasapps/) if you have not previously configured applications on your system.
+
+## Introduction and Preparation
+
+TrueNAS SCALE lets you run apps using Docker.
+
+There are two supported methods of installing Jellyfin on your TrueNAS SCALE server:
+
+1. Custom App
+2. Community 3rd Party App provided by ix-systems
+
+Both methods use the official Jellyfin-provided Docker image.
+This documentation will cover both install methods.
+
+You can configure environment variables at any time after deploying the application.
+
+It's recommend to set the TrueNAS SCALE App system to a storage pool comprised of SSDs.
+
+---
+
+### Datasets & Jellyfin
+
+The official Jellyfin Docker image internally creates the following necessary directories:
+
+1. cache
+2. config
+3. config/transcodes
+
+If you use the community app, you can allow SCALE to create datasets for the directories Jellyfin requires automatically during app installation.
+You can also choose to create a static **transcodes** dataset or use temporary storage on the disk or in memory (system RAM) for transcoding.
```

Comment:
Please tell users using ram is not a good idea as it can easily run out and cause transcodes to fail. SSD is recommended as using HDD might cause slowdowns